python/requirements-build.txt

```diff
@@ -0,0 +1,8 @@
+six>=1.0.0
```

Comment:
What about using some of the other requirement files here? You should be able to reference them with
`-r requirements-wheel.txt`